src/main/java/teammates/common/util/Const.java

```diff
@@ -359,9 +359,8 @@ private Const() {
                 "This feedback session is not published as it is private and only visible to you.";
 
         public static final String FEEDBACK_SESSION_INPUT_TIMEZONE =
-                "You should not need to change this as your timezone is auto-detected. <br><br>"
-                + "However, note that daylight saving is not taken into account i.e. if you are in UTC -8:00 and there is "
-                + "daylight saving, you should choose UTC -7:00 and its corresponding timings.";
+                "You should not need to change this as your time zone is auto-detected. "
+                + "Daylight saving time is supported.";
```

Comment:
`Note that TEAMMATES now (since April 2018) automatically adjusts to match daylight saving adjustments in your area.`?

Reply:
Can I omit the `(since April 2018)` part? I feel that the product history should not form a permanent part of the UI, and that the modal dialog is sufficient to inform existing users.

Also, is this phrasing okay? I modified it slightly so that it encompasses offset changes not caused by DST, e.g. when the zone rules are changed by governments:
`You should not need to change this as your time zone is auto-detected. TEAMMATES automatically adjusts to match the current time offset in your area, including clock changes due to daylight saving time.`

Source: "Clock changes" is a term used in https://www.timeanddate.com/time/dst/events.html

Reply:
Looks good 👍